resources/views/dashboard/controllers/events/view.blade.php

```diff
@@ -296,6 +333,11 @@
                                                 {!! Form::text('end_time1', null, ['placeholder' => $event->end_time, 'class' => 'form-control']) !!}
                                             </div>
                                         </div>
+                                        <div class="row pt-2">
+                                            <div class="col-sm-11">
+                                                {!! Form::textarea('remarks', null, ['placeholder' => 'Enter any relevant remarks here...', 'class' => 'form-control textarea-no-resize']) !!}
```

Comment:
Several comments here:
1. I'd like to see more relevant/descriptive placeholder text. Please work with Dhagash on this and come up with something that better describes the type of input that he's hoping to garner from this form element.
2. You need to add a maxlength parameter on this form element - exceeding the maxlength currently causes the form to fail without an error. See this article for the blade way of doing this: https://stackoverflow.com/questions/27525832/laravel-form-inputs-how-to-set-maxlength
3. The remarks box is HUGE (height) on my screen. Can we constrain this to about 3 rows with a scrollbar, please?